src/endpoints/translate.js

```diff
@@ -61,39 +64,88 @@ router.post('/libre', jsonParser, async (request, response) => {
     }
 });
 
+
 router.post('/google', jsonParser, async (request, response) => {
     try {
         const { generateRequestUrl, normaliseResponse } = require('google-translate-api-browser');
-        const text = request.body.text;
+        let text = request.body.text;
         const lang = request.body.lang;
 
         if (!text || !lang) {
             return response.sendStatus(400);
         }
 
-        console.log('Input text: ' + text);
+        console.log('Input text:', text);
+        //console.log('----------');
+
+        // Apply formatting only if the translation language is Russian
+        if (lang === 'ru') {
+             // Replace quotation marks and asterisks with special markers before translating
+            const openQuote = '__OPEN_QUOTE__ ';
```

Comment:
Some Asian languages also use specific quotes. Also how do you make sure that these placeholders don't leak into the output?